src/interpreter/plugin/SimpleArithmertic.ts

```diff
@@ -0,0 +1,217 @@
+import {ArgumentTypes, FunctionPlugin} from './FunctionPlugin'
+
+export class SimpleArithmerticPlugin extends  FunctionPlugin {
+  public static implementedFunctions = {
+    'HF.ADD': {
+      method: 'add',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.CONCAT' : {
+      method: 'concat',
+      parameters: [
+        { argumentType: ArgumentTypes.STRING },
+        { argumentType: ArgumentTypes.STRING },
+      ],
+    },
+    'HF.DIVIDE': {
+      method: 'divide',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.EQ': {
+      method: 'eq',
+      parameters: [
+        { argumentType: ArgumentTypes.NOERROR },
+        { argumentType: ArgumentTypes.NOERROR },
+      ]
+    },
+    'HF.GT': {
+      method: 'gt',
+      parameters: [
+        { argumentType: ArgumentTypes.NOERROR },
+        { argumentType: ArgumentTypes.NOERROR },
+      ]
+    },
+    'HF.GTE': {
+      method: 'gte',
+      parameters: [
+        { argumentType: ArgumentTypes.NOERROR },
+        { argumentType: ArgumentTypes.NOERROR },
+      ]
+    },
+    'HF.LT': {
+      method: 'lt',
+      parameters: [
+        { argumentType: ArgumentTypes.NOERROR },
+        { argumentType: ArgumentTypes.NOERROR },
+      ]
+    },
+    'HF.LTE': {
+      method: 'lte',
+      parameters: [
+        { argumentType: ArgumentTypes.NOERROR },
+        { argumentType: ArgumentTypes.NOERROR },
+      ]
+    },
+    'HF.MINUS': {
+      method: 'minus',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.MULTIPLY': {
+      method: 'multiply',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.NE': {
+      method: 'ne',
+      parameters: [
+        { argumentType: ArgumentTypes.NOERROR },
+        { argumentType: ArgumentTypes.NOERROR },
+      ]
+    },
+    'HF.POW': {
+      method: 'pow',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.UMINUS': {
+      method: 'uminus',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.UNARY_PERCENT': {
+      method: 'unary_percent',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+    'HF.UPLUS': {
+      method: 'uplus',
+      parameters: [
+        { argumentType: ArgumentTypes.NUMBER },
+      ],
+    },
+  }
+
+  public add(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
+    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.ADD'),
+      (a: number, b: number) => this.interpreter.arithmeticHelper.addWithEpsilon(a, b)
+      )
```

Comment:
If passing ref will work, we can do the same here
```suggestion
    return this.runFunction(ast.args, formulaAddress, this.metadata('HF.ADD'), this.interpreter.arithmeticHelper.addWithEpsilon)
```